server/__test__/singup.test.ts

```diff
@@ -5,33 +5,35 @@ import { seedDatabase } from '../db';
 
 beforeAll(() => seedDatabase());
 
-describe('Testing the signup route.', () => {
-  it('Testing the success path, the controller should return 201 stats code.', done => {
+describe('Testing the signup route', () => {
+  it('test the path, controller should return 201 stats code', done => {
     supertest(app)
       .post('/api/v1/auth/register')
       .send({
-        username: 'meee',
-        password: 'root123',
-        roleId: 1,
+        username: 'joe Does1',
+        password: '123456',
+        roleId: 2,
       })
+      .expect(201)
       .end((err, res) => {
         if (err) return done(err);
-        expect(res.status).toBe(201);
+
+        expect(res.body.msg).toBe('the user has been created successfully');
```

Comment:
The message in the response body is not the same as this message in the test, so it's failing.
Change the message to make it pass.

---
![Screenshot from 2023-06-17 22-40-52](https://github.com/GSG-G13/MenuMingle/assets/77394697/82cf3313-3417-42eb-84c2-edd28b5af9cf)